protocol-units/da/m1/util/src/config/common.rs

```diff
@@ -1,5 +1,20 @@
+use alloy::signers::local::PrivateKeySigner;
 use celestia_types::nmt::Namespace;
+/*use ecdsa::{
+	elliptic_curve::{
+		generic_array::ArrayLength,
+		ops::Invert,
+		point::PointCompression,
+		sec1::{FromEncodedPoint, ModulusSize, ToEncodedPoint},
+		subtle::CtOption,
+		AffinePoint, CurveArithmetic, FieldBytesSize, PrimeCurve, Scalar,
+	},
+	hazmat::{DigestPrimitive, SignPrimitive, VerifyPrimitive},
+	SignatureSize,
+};*/
```

Comment:
Just remove this.

Reply:
I need to come back to it for serialization and deserialization.